RELEASE-NOTES.md

```diff
@@ -0,0 +1,51 @@
+# node_package 3.0.0
+## Security Improvements
+### Introduction
+### Security Advisory dated March 1, 2016
+It was [recently reported](http://docs.basho.com/riak/latest/community/product-advisories/codeinjectioninitfiles/) that, if a user could gain access to the `riak` user (or, in node_package parlance, the `package_install_user`), that use would then have write access to init scripts that are generally run as `root`, exposing an escalation of privileges attack where said use could then get the `root` user to execute a script that could allow the original user to become `root` on the system.
```

Comment:
use -> user (also that -> the to avoid the that that problem): "...`package_install_user`), the user would then have write access to init scripts that are generally run as `root`, exposing an escalation of privileges attack where said user could then ..."